src/script/main/app.js

```diff
@@ -290,29 +282,30 @@ class App {
    * @returns {undefined} No return value
    */
   initApp(isReload = this._isReload()) {
+    const loadingView = new LoadingViewModel(this.repository.user.self);
```

Comment:
Nice benefit, since the loadingView is never used anywhere else, we just declare it here and then the garbage collector will take care of removing it from memory. 

(before it stayed in memory forever)